components/core/Settings/Jumper.js

```diff
@@ -0,0 +1,412 @@
+import * as UserBehaviors from "~/common/user-behaviors";
+
+import { css } from "@emotion/react";
+import { useSettingsContext } from "~/components/core/Settings/Provider";
+import { AnimatePresence, motion } from "framer-motion";
+
+import ProfilePhoto from "~/components/core/ProfilePhoto";
+
+import { Input } from "~/components/system/components/Input";
+import { Textarea } from "~/components/system/components/Textarea";
+import { ButtonPrimary, ButtonSecondary } from "~/components/system/components/Buttons";
+import * as Type from "~/components/system/components/Typography";
+
+const STYLES_FLEX_COLUMN = `
+  display: flex;
+  flex-direction: column;
+  position: static;
+  align-items: flex-start;
+`;
+
+const STYLES_JUMPER_HEADER = css`
+  ${Styles.HORIZONTAL_CONTAINER_CENTERED};
+  padding: 17px 20px 15px;
+`;
+
+const STYLES_JUMPER_OVERLAY = (theme) => css`
+  position: fixed;
+  width: 100%;
+  height: 100%;
+  z-index: ${theme.zindex.jumper};
+  @supports ((-webkit-backdrop-filter: blur(75px)) or (backdrop-filter: blur(75px))) {
+    -webkit-backdrop-filter: blur(75px);
+    backdrop-filter: blur(75px);
+    background-color: ${theme.semantic.bgBlurLightTRN};
+  }
+`;
+
+const STYLES_JUMPER_DISMISS_BUTTON = (theme) => css`
+  ${Styles.BUTTON_RESET};
+  color: ${theme.semantic.textGray};
+`;
+
+const STYLES_JUMPER_MAIN = css`
+  ${Styles.HORIZONTAL_CONTAINER};
+  justify-content: space-between;
+  align-items: flex-start;
+  width: 100%;
+  height: 348px;
+  order: 1;
+`;
+
+const STYLES_JUMPER_SIDEBAR = css`
+  ${Styles.HORIZONTAL_CONTAINER};
+  align-items: flex-start;
+  width: 190px;
+  height: 100%;
+  padding: 20px 0px;
+  border-right: 1px solid ${Constants.system.grayLight5};
+`;
+
+const STYLES_JUMPER_CONTENT = css`
+  ${STYLES_FLEX_COLUMN}
+  align-items: flex-start;
+  width: 100%;
+  height: 340px;
+  overflow: scroll;
+  margin: 0px 20px;
+  padding-bottom: 160px;
+  padding-top: 20px;
+  ::-webkit-scrollbar {
+    width: 0;
+    background: transparent; 
+  }
+`;
+
+const STYLES_ACCOUNT = css`
+  ${STYLES_FLEX_COLUMN}
+  align-items: center;
+  width: 100%;
+`;
+
+const STYLES_ACCOUNT_USER = css`
+  ${STYLES_FLEX_COLUMN}
+  align-items: center;
+  margin: 10px 0px;
+`;
+
+const STYLES_ITEM_CONTAINER = css`
+  ${STYLES_FLEX_COLUMN}
+  width: 100%;
+  height: 76px;
+  margin-top: 24px;
+`;
+
+const STYLES_ITEM_HEADER = css`
+  font-size: 12px;
+  margin-bottom: 8px;
+  color: ${Constants.system.gray};
+`;
+
+const STYLES_AVATAR_BOX = css`
+  ${STYLES_FLEX_COLUMN}
+  width: 100%;
+  height: 76px;
+`;
+
+const STYLES_AVATAR_UPLOAD = css`
+  ${Styles.HORIZONTAL_CONTAINER};
+  align-items: flex-end;
+`;
+
+const STYLES_EDIT_INFO_FOOTER = css`
+  display: flex;
+  position: absolute;
+  bottom: 0;
+  right: 0;
+  width: 448px;
+  height: 50px;
+  padding: 12px 20px;
+  background-color: ${Constants.system.white};
+  border-radius: 0px 0px 16px 0px;
+  border-top: 1px solid ${Constants.system.grayLight5};
```

Comment:
in 99% of the case, use semantic color rather than system color. 
This would make things easier if we ever go dark mode in the future.
I think the one should be `${Constants.semantic.borderGrayLight}`